### PR TITLE
Fix total_sum calculation and fix shipping check

### PR DESCRIPTION
**Pull request by mikakoi**

### CODE REVIEW

File: logitrail-woocommerce.php [hunk 1]
```diff
@@ -802,9 +817,9 @@ function logitrail_barcode_save($post_id){
 
     function enable_logitrail_shipping_save($post_id) {
         if( !empty($_POST['enable_logitrail_shipping']) ) {
-            update_post_meta( $post_id, 'enable_logitrail_shipping', esc_attr( $_POST['enable_logitrail_shipping'] ) );
+            update_post_meta( $post_id, 'logitrail_enable_shipping', esc_attr( $_POST['logitrail_enable_shipping'] ) );
```
Comment (Orppa) [inline logitrail-woocommerce.php]: Some actual reason for this change? And now setting and function name don't match..

File: logitrail-woocommerce.php [hunk 2]
```diff
@@ -1022,19 +1037,34 @@ function update_product() {
...
-        // Disable shipping on some product types
-        if ( in_array($product->get_type(), array("bundle")) ) {
+    /**
+     * @param WC_Product $product
+     * @param bool $ignore_bundled
+     * @return bool
+     */
+    public function logitrail_is_virtual($product, $ignore_bundled = false) {
+        $test = wc_pb_get_bundled_product_map($product);
+        // We don't want to ship virtual products, but bundled products are virtual so exclude them
```
Comment (Orppa) [inline logitrail-woocommerce.php]: Doesn't this just tell is the product virtual or not? It doesn't make any decision about excluding anything?

Comment (mikakoi) [reply logitrail-woocommerce.php]: if the product is a bundled product, the function returns false

Comment (Orppa) [reply logitrail-woocommerce.php]: Yes, false is a boolean value answering to the question the function name proposes. It doesn't say anything about exclusion, which is a decision made elsewhere based on the return value of this function.

File: logitrail-woocommerce.php [hunk 3]
```diff
@@ -1022,19 +1038,33 @@ function update_product() {
...
-        $product = wc_get_product( $product_id );
-        // Disable shipping on some product types
-        if ( in_array($product->get_type(), array("bundle")) ) {
+    /**
+     * @param WC_Product $product
+     * @param bool $ignore_bundled
+     * @return bool
+     */
+    public function logitrail_is_virtual($product, $ignore_bundled = false) {
+        // We don't want to ship virtual products, but bundled products are virtual so exclude them
```
Comment (Orppa) [inline logitrail-woocommerce.php]: "We don't want to ship virtual products, so we check is the product really virtual, or just part of bundle and marked virtual for this"

**Review by Orppa — APPROVED** [review]

**Pull request merged**